25.md

```diff
@@ -33,7 +28,8 @@ The last `p` tag MUST be the `pubkey` of the event being reacted to.
 
 The `a` tag MUST contain the coordinates (`kind:pubkey:d-tag`) of the replaceable being reacted to.
 
-The reaction event MAY include a `k` tag with the stringified kind number of the reacted event as its value.
+The reaction event SHOULD include a `k` tag with the stringified kind number of the reacted event as
+its value, and a `c` tag with the reaction's `content` so that reactions can be indexed and filtered.
```

Comment:
I think having `c` as a sentiment category with values  [+, -, 0]  for positive, negative and neutral is more useful than simply copying the `content`.

Reply:
Where does this mapping come from? If from the client, it doesn't solve anything about assuming user intention. If from the user, then it adds UX friction. If your users want to explain their emojis, that's fine, I just don't think that would be widely adopted.